_data/meltano/utilities/superset_ext/apache.yml

```diff
@@ -0,0 +1,145 @@
+name: superset_ext
+namespace: superset
+label: Superset (EDK preview)
+domain_url: https://superset.apache.org/
+description: A modern, enterprise-ready business intelligence web application.
+docs: https://docs.meltano.com/guide/analysis
+repo: https://github.com/apache/superset
+maintenance_status: active
+variant: apache
+pip_url: apache-superset==2.0.0 flask==2.0.3 werkzeug==2.0.3 jinja2==3.0.1 wtforms==2.3.3 git+https://github.com/meltano/superset-ext.git@main
+settings:
+- name: ui.bind_host
+  label: UI Bind Host
+  value: "0.0.0.0"
+  description: Host used by `meltano invoke superset:ui`. Used in the `gunicorn` `--bind` option along with `ui.port`.
+- name: ui.port
+  label: UI Port
+  value: 8088
+  description: Port used by `meltano invoke superset:ui`. Used in the `gunicorn` `--bind` option along with `ui.bind_host`.
+- name: ui.timeout
+  label: UI Timeout
+  value: 60
+  description: Timeout in sections, used by `meltano invoke superset:ui`. Corresponds to the `gunicorn` `--timeout` option
+- name: ui.workers
+  label: UI Workers
+  value: 4
+  description: Workers used by `meltano invoke superset:ui`. Corresponds to the `gunicorn` `--workers` option
+- name: SUPERSET_CONFIG_PATH
+  label: Superset config path
+  description: The path to where the Superset config file is located.
+  value: $MELTANO_PROJECT_ROOT/superset/superset_config.py
```

Comment:
Airflow we ended up organizing under a `$MELTANO_PROJECT_ROOT/orchestrate/` , do we want to stash superset under something similar?